src/library/library.cpp

```diff
@@ -706,3 +706,13 @@ LibraryTableModel* Library::trackTableModel() const {
 
     return m_pMixxxLibraryFeature->trackTableModel();
 }
+
+void Library::importPlaylistFromFile(QString playlistFile) {
+    int playlistId;
+    if (!playlistFile.isNull()) {
```

Comment:
You can move this up and return early if `playlistFile.isEmpty()` (which also covers `isNull()`).